.github/workflows/test.yml

```diff
@@ -51,6 +52,28 @@ jobs:
       - name: Run tests
         run: pytest
 
+      - name: Test CLI
+        run: graphium --help
+
+      - name: Test building the doc
+        run: mkdocs build
+  
+  codecov:
```

Comment:
Any reason for having a new section here instead of having just a coverage step above?

Reply:
It seems best to have everything above especially cuz the test matrix would have to be defined twice.

Reply:
Nope, that's true. I was trying to make it possible to use the full coverage based on the two env tests, but I think this is a simpler solution.